internal/header/header.go

```diff
@@ -0,0 +1,37 @@
+// This source file is part of the EdgeDB open source project.
+//
+// Copyright 2020-present EdgeDB Inc. and the EdgeDB authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package header
+
+import "encoding/binary"
+
+const (
+	// AllowCapabilities tells the server what capabilities it should allow.
+	AllowCapabilities        = 0xFF04
```

Comment:
```suggestion
	AllowCapabilitiesHeader        = 0xFF04
```

Reply:
The module name `header` will always be present when this is used in the edgedb package. Adding `Header` is redundant.